reo/nested_outputs.py

```diff
@@ -82,8 +82,35 @@ def list_of_string(input):
             },
 
             "Site": {
+              "year_one_emissions_lb_C02": {
+                "type": int,
+                "description": "Total equivalent pounds of carbon dioxide emitted from the site in the first year.",
+                "units": "hours"
+              },
+
+              "total_emissions_lb_C02": {
+                "type": int,
+                "description": "Total equivalent pounds of carbon dioxide emitted from the site over all years in the analysis period.",
+                "units": "hours"
+              },
+              "year_one_emissions_bau_lb_C02": {
+                "type": int,
+                "description": "Total equivalent pounds of carbon dioxide emitted from the site use in the first year in the BAU case.",
+                "units": "hours"
+              },
+
+              "total_emissions_bau_lb_C02": {
+                "type": int,
+                "description": "Total equivalent pounds of carbon dioxide emitted from the site over all years in the analysis period in the BAU case.",
+                "units": "hours"
+              },
 
               "LoadProfile": {
+                "year_one_electric_load_series_kw_bau": {
```

Comment:
see previous comment: I think that this output is redundant

Reply:
As above, this has been removed via commit 036435b22aada8a3e485ec07cdd319979db22321